mean/modules/patients/server/models/patient.server.model.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var mongoose = require('mongoose'),
+  Schema = mongoose.Schema;
+
+/**
+ * Article Schema
+ */
+var PatientSchema = new Schema({
+  created: {
```

Comment:
Does it matter when the patient was created?  Or is there something you could put here that would be more descriptive, like the date of the first visit, or last visit.